azkaban-web-server/src/test/java/azkaban/imagemgmt/services/ImageTypeServiceImplTest.java

```diff
@@ -75,9 +77,25 @@ public void testCreateImageType() throws Exception {
     Assert.assertEquals(100, imageTypeId);
   }
 
+  @Test
+  public void testFindImageTypeWithOwnersByName() throws Exception {
+    final ImageType imageType = new ImageType();
+    when(this.imageTypeDao.getImageTypeWithOwnershipsByName(any(String.class))).thenReturn
+        ((Optional<ImageType>) Optional.of(imageType));
+    this.imageTypeService.findImageTypeWithOwnersByName("anyString");
+  }
```

Comment:
Are we missing an assert here somewhere?

Reply:
I can make this test a little better, thank you!